### ISSUE TITLE
UnusedPrivateElements.UnusedMethod false positive when calling private method on the same class

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#211]
Following code leads to `SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod` error.

```php
class Foo
{

    public static function create()
    {
        $foo = new Foo(); // works with self
        $foo->privateFn();
    }

    private function privateFn() {}

}
```

> Class Foo contains unused private method privateFn().

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/3c1826af9ef1cc2fbc70c51299549079b31d1e90.
